build.sbt

```diff
@@ -1,13 +1,33 @@
 // Copyright: 2017 https://github.com/cakesolutions/sbt-cake/graphs
 // License: http://www.apache.org/licenses/LICENSE-2.0
 
+import de.heikoseeberger.sbtheader.FileType
+import de.heikoseeberger.sbtheader.HeaderPlugin.autoImport.HeaderFileType
+
 organization := "net.cakesolutions"
 
+// scalastyle:off magic.number
+
 name := "sbt-cake"
 sbtPlugin := true
 
 sonatypeGithub := ("cakesolutions", "sbt-cake")
+startYear := Some(2017)
 licenses := Seq(Apache2)
+headerLicense := Some(
+  HeaderLicense.Custom(
+    """|Copyright: 2017 https://github.com/cakesolutions/sbt-cake/graphs
+     |License: http://www.apache.org/licenses/LICENSE-2.0
+     |""".stripMargin
+  )
+)
+headerMappings :=
+  headerMappings.value ++
+  Map(
+    FileType("sbt") -> HeaderCommentStyle.CppStyleLineComment,
+    HeaderFileType.java -> HeaderCommentStyle.CppStyleLineComment,
```

Comment:
I observed the above when I ran `sbt headerCreate` within sbt-cake. Hence why I felt it necessary to alter things here.